tla-bmcmt/src/test/scala/at/forsyte/apalache/tla/bmcmt/stratifiedRules/aux/oracles/TestSparseOracle.scala

```diff
@@ -70,7 +70,8 @@ class TestSparseOracle extends AnyFunSuite with BeforeAndAfterEach with Checkers
   // which should have its own tests
 
   // We cannot test getIndexOfChosenValueFromModel without running the solver
-  test("getIndexOfChosenValueFromModel recovers the index correctly") {
+  // Ignored until we figure out why it's killing GH CLI
+  ignore("getIndexOfChosenValueFromModel recovers the index correctly") {
```

Comment:
Can we
- [ ] refer to the relevant issue here, and
- [ ] leave a note under that issue to (re-)enable these tests?